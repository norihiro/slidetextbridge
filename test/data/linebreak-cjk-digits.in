令和7年6月7日 (土) から同年7月7日 (月) までの間、意見募集を行いました。

７月の日本の月平均気温は、統計開始（1898年）以降の７月として最も高くなりました。また、東北日本海側と北陸地方では降水量が記録的に少なくなりました。
