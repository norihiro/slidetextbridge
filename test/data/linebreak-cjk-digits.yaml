steps:
  - type: stdin

  - type: normalize

  - type: linebreak
    split_long_line: 10
    split_nowrap: "、。」』,."
    joined_column_max: 8

  - type: stdout
    page_delimiter: "\n"
