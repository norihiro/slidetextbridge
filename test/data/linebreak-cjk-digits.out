令和7年6月
7日 (土) 
から同年7
月7日 (月)
までの間、
意見募集を
行いました。
7月の日本
の月平均気
温は、統計
開始(1898
年)以降の7
月として最
も高くなり
ました。ま
た、東北日
本海側と北
陸地方では
降水量が記
録的に少な
くなりまし
た。
